public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/descriptions.tsx

```diff
@@ -0,0 +1,32 @@
+import { RuleFormType } from '../../../types/rule-form';
+
+type FormDescriptions = {
+  sectionTitle: string;
+  helpLabel: string;
+  helpContent: string;
+  helpLink: string;
+};
+
+export const DESCRIPTIONS: Record<RuleFormType, FormDescriptions> = {
+  [RuleFormType.cloudRecording]: {
+    sectionTitle: 'Define recording rule',
+    helpLabel: 'Define your recording rule',
+    helpContent:
+      'Pre-compute frequently needed or computationally expensive expressions and save their result as a new set of time series.',
+    helpLink: '',
+  },
+  [RuleFormType.grafana]: {
+    sectionTitle: 'Define query and alert condition',
+    helpLabel: 'Define query and alert condition',
+    helpContent:
+      'An alert rule consists of one or more queries and expressions that select the data you want to measure. Define queries and/or expressions and then choose one of them as the alert rule condition. This is the threshold that an alert rule must meet or exceed in order to fire. For more information on queries and expressions, see Query and transform data.',
+    helpLink: 'https://grafana.com/docs/grafana/latest/panels-visualizations/query-transform-data/',
+  },
+  [RuleFormType.cloudAlerting]: {
+    sectionTitle: 'Define query and alert condition',
+    helpLabel: 'Define query and alert condition',
+    helpContent:
+      'An alert rule consists of one or more queries and expressions that select the data you want to measure. Define queries and/or expressions and then choose one of them as the alert rule condition. This is the threshold that an alert rule must meet or exceed in order to fire. For more information on queries and expressions, see Query and transform data.',
+    helpLink: 'https://grafana.com/docs/grafana/latest/panels-visualizations/query-transform-data/',
```

Comment:
Not sure how this works.  If `cloudAlerting` is for Cloud instances, then, it could be more appropriate to link to the Cloud docs:
https://grafana.com/docs/grafana-cloud/visualizations/panels-visualizations/query-transform-data/

Still learning for any valuable review but the rest sounds good to me.

Reply:
The term is a bit misleading here, we call these `data source managed alerts` now but the code still refers to those as `cloudAlerting` :)